website/docs/d/netapp_volume.html.markdown

```diff
@@ -43,6 +43,8 @@ The following attributes are exported:
 
 * `location` - The Azure Region where the NetApp Volume exists.
 
+* `mount_ip_addresses` - A list of IPv4 Addresses which should be used to mount the volume.
```

Comment:
this has also been added to the Resource - so can we document this field there too?

Reply:
I didn't add this field "mount_ip_addresses" to the resource file. I just added this to data source file. So I think we don't need to document this field to resource md file.

Reply:
Is there a reason we are only adding it to the datasource and not the resource too? @neil-yechenwei

Reply:
You can find user's usage preference from (https://github.com/terraform-providers/terraform-provider-azurerm/issues/5416), seems they want use data source to retrieve this property value.